World_Simulation/auto_gauges.py

```diff
@@ -110,12 +106,12 @@ def near_shoreline(storm_x, storm_y, shoreline):
 def distance(x1, y1, x2, y2):
     
     # convert to radians
-    x1 = x1*math.pi/180
-    x2 = x2*math.pi/180
-    y1 = y1*math.pi/180
-    y2 = y2*math.pi/180
+    x1 = x1*np.pi/180
+    x2 = x2*np.pi/180
+    y1 = y1*np.pi/180
+    y2 = y2*np.pi/180
     
-    distance = 2*6378.137*np.arcsin(math.sqrt(math.sin((y2-y1)/2)**2 + math.cos(y1)*math.cos(y2)*(math.sin((x2-x1)/2)**2)))
+    distance = 2*6378.137*np.arcsin(np.sqrt(np.sin((y2-y1)/2)**2 + np.cos(y1)*np.cos(y2)*(np.sin((x2-x1)/2)**2)))
     
     return distance
 
```

Comment:
You should use the function available in GeoClaw that does this.

Reply:
There's a haversine function available there that will allow you not to rely on constants.

Reply:
It's in `clawpack.geoclaw.util`.

Reply:
Hi Dr. Mandli! Thank you, we will look into it and update it soon.